packages/gasket-plugin-https-proxy/README.md

```diff
@@ -104,6 +113,22 @@ gasket.actions.startProxyServer();
 
 ## Lifecycles
 
+### preboot
```

Comment:
I think something as essential as "things I want to do before my server starts" would be misplaced being part of `@gasket/plugin-https-proxy` which seems like a plugin that would not be necessarily a universal thing but more like a weird plugin we have to work around next.js specificities. I was thinking it should be part of `@gasket/plugin-https` instead (which would be better named `@gasket/plugin-web-server`).